### DISCUSSION TITLE
Grademelo0n

### DISCUSSION THREAD

**Discussion by Jshap06** [PurelyAnecdotal/gradecompass#55] [General]
What's up, I run my own thing like this out of Maryland called Grademelon at Grademelon.org
I have the API working, which you seem not to have. Wanna connect?

**Comment by Jshap06** [comment]
At least not for my county, I got a fix for that if u want it, clearly it still works for ur county ig.

**Comment by Jshap06** [comment]
For a lot of counties u need an auth cookie now for api requests to go through

**Comment by PurelyAnecdotal** [comment]
Hi, if you want to DM my discord is the same as my GitHub username

**Comment by Jshap06** [comment]
This is some quality work btw
